java/client/src/org/openqa/selenium/interactions/Actions.java

```diff
@@ -380,7 +380,7 @@ public Actions moveToElement(WebElement target, int xOffset, int yOffset) {
 
     // Of course, this is the offset from the centre of the element. We have no idea what the width
     // and height are once we execute this method.
-    LOG.info("When using the W3C Action commands, offsets are from the center of element");
+    LOG.finest("When using the W3C Action commands, offsets are from the center of element");
```

Comment:
Again, the existing level is correct --- we're letting users know about something that might cause there tests to fail.